src/main/scala/scalismo/statisticalmodel/StatisticalVolumeIntensityModel.scala

```diff
@@ -0,0 +1,100 @@
+package scalismo.statisticalmodel
+
+import breeze.linalg.DenseVector
+import scalismo.common._
+import scalismo.geometry._
+import scalismo.mesh._
+import scalismo.utils.Random
+
+import scala.reflect.ClassTag
+import scala.reflect.runtime.universe.TypeTag
+
+trait StatisticalVolumeIntensityModel[S] {
+
+  def referenceMesh: TetrahedralMesh3D
+
+  def referenceMeshField: ScalarVolumeMeshField[S]
+
+  def shape: StatisticalVolumeMeshModel
+
+  def intensity: DiscreteLowRankGaussianProcess[_3D, UnstructuredPointsDomain[_3D], S]
+
+  def landmarks: Map[String, Landmark[_3D]] = Map.empty[String, Landmark[_3D]]
```

Comment:
I don't think we need landmarks in the model. This makes sense for faces (in the MoMO implementation), as we are always modelling faces. But for most shapes we could not define a canonical set of (named) landmarks. And if we can, we can always store them in a separate file.

Reply:
This makes sense. I will remove them.